io/plugins/eu.esdihumboldt.hale.io.jdbc.postgresql.test/src/eu/esdihumboldt/hale/io/jdbc/postgresql/test/PostDataTypesTest.groovy

```diff
@@ -0,0 +1,137 @@
+
+
+
+
+/*
+ * Copyright (c) 2015 Data Harmonisation Panel
+ * 
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Contributors:
+ *     Data Harmonisation Panel <http://www.dhpanel.eu>
+ */
+
+package eu.esdihumboldt.hale.io.jdbc.postgresql.test
+
+import static org.junit.Assert.*
+
+import java.sql.Date
+import java.sql.Timestamp
+
+import org.junit.Test
+
+import com.vividsolutions.jts.geom.Coordinate
+import com.vividsolutions.jts.geom.Geometry
+import com.vividsolutions.jts.geom.GeometryFactory
+
+import eu.esdihumboldt.hale.common.instance.geometry.DefaultGeometryProperty
+import eu.esdihumboldt.hale.common.instance.geometry.impl.CodeDefinition
+import eu.esdihumboldt.hale.common.instance.groovy.InstanceBuilder
+import eu.esdihumboldt.hale.common.instance.model.InstanceCollection
+import eu.esdihumboldt.hale.common.schema.model.Schema
+import eu.esdihumboldt.hale.common.schema.model.TypeDefinition
+import eu.esdihumboldt.hale.io.jdbc.test.AbstractDBTest
+import eu.esdihumboldt.hale.io.jdbc.test.DBConfigInstance
+import groovy.sql.Sql
+
+
+/**
+ * SQL type and binding test for postgresql database
+ * @author sameer sheikh
+ */
+public class PostDataTypesTest extends AbstractDBTest {
+
+	public static final String query = ''' CREATE TABLE employees
+
+    ( employee_id    INTEGER 
+    , first_name    VARCHAR
+	, handicaped     bool
+    , decimal_test   DECIMAL
+    , dec_test       NUMERIC   
+	, b_array		 BYTEA
+	, salary		 MONEY
+    , geometry_test  geometry(LineString, 4326)
+    , hire_date      DATE  NOT NULL
+    , last_login     TIMESTAMP
+	, share_price    FLOAT
+	, pers_msg       TEXT
+    ,  CONSTRAINT lines_pkey PRIMARY KEY (EMPLOYEE_ID)
+    ) ;'''
+	private final Map<String, Class<?>> map = createMap();
+	public PostDataTypesTest(){
+		super(new DBConfigInstance("postgis"))
+	}
+	/**
+	 * Test for checking sql type and binding
+	 */
+	@Test
+	void testDataTypes(){
+		Sql s = new Sql(waitForConnection());
+		try{
+			s.execute query;
+		}
+		finally{
+			s.close();
+		}
+		Schema schema = readSchema()
+		checkBidingAndSqlType(schema,map);
```

Comment:
You fixed the spelling at one point, but not here - which results in the test failing.